convert/src/vertex_to_bd/badgerdoc_format/plain_text_converter.py

```diff
@@ -0,0 +1,78 @@
+from typing import List
+
+from ..models import BadgerdocToken, Offset, Page, PageSize
+
+DEFAULT_PAGE_BORDER_OFFSET = 15
+
+
+def generate_chunks(text: str, n=70) -> List[str]:
```

Comment:
May you rename `n` to more "meaningful" name =)
And don't forget about typing

Reply:
fixed